cmd/notify-nats.go

```diff
@@ -52,7 +53,7 @@ func (n *natsNotify) Validate() error {
 	if !n.Enable {
 		return nil
 	}
-	if _, err := checkURL(n.Address); err != nil {
+	if _, _, err := net.SplitHostPort(n.Address); err != nil {
```

Comment:
Yes port should be provided..  existing configs are broken anyways because of the url usage.